src/index.js

```diff
@@ -109,30 +154,24 @@ export default class Luxafor {
    * @returns {object} Instance
    */
   exec() {
-
```

Comment:
This was a bug, not being able to do commands without specifying any ledPositions.